### PR TITLE
Updated node-gather API and fixed minor image bug

### PR DESCRIPTION
**Pull request by alevitter**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1816778 by updating the api.
Also fixes a bug where the images directory was not being created

Signed-off-by: Avram Levitter <alevitte@redhat.com>

### CODE REVIEW

File: collection-scripts/gather_images [hunk 1]
```diff
@@ -7,6 +7,7 @@ BASE_COLLECTION_PATH="/must-gather"
 IMAGES_PATH=${BASE_COLLECTION_PATH}/cluster-scoped-resources/images
 NAMESPACE_PATH=${BASE_COLLECTION_PATH}/namespaces
 
+mkdir ${BASE_COLLECTION_PATH}/cluster-scoped-resources
```
Comment (masayag) [inline collection-scripts/gather_images]: pls use `mkdir -p`

Comment (masayag) [reply collection-scripts/gather_images]: you can also pull ${BASE_COLLECTION_PATH}/cluster-scoped-resources into variable and use it for IMAGES_PATH prefix

Comment (alevitter) [reply collection-scripts/gather_images]: Done

**Comment by masayag** [comment]
@alevitter what's the reason for travis failure ?

**Comment by alevitter** [comment]
> @alevitter what's the reason for travis failure ?

The travis is currently not set up to not run on a pull request. It fails because the pull request doesn't have access to the user/password for Quay, so the push is unsuccessful.

**Comment by masayag** [comment]
@alevitter is it safe to merge like that ?

**Pull request merged**